### ISSUE TITLE
nevalidní html (Bugzilla Bug 207)

### ISSUE THREAD

**Issue by abclinuxu** [abclinuxu/abclinuxu#207]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 207
Date: 2005-08-23 00:31:11 -0400
From: hajma &lt;<tropikhajma@seznam.cz>&gt;
To:   Robert Krátký &lt;<rk@stickfish.cz>&gt;

Duplicates:   #527
Depends on:   #527
Last updated: 2006-12-19 13:50:18 -0500

## Comment 522
Date: 2005-08-23 00:31:11 -0400
From: hajma &lt;<tropikhajma@seznam.cz>&gt;

hned na www.abclinuxu.cz mi html validator (extension ve FF) ukazuje tato varování:

line 33 column 209 - Warning: '<' + '/' + letter not allowed here

line 33 column 215 - Warning: '<' + '/' + letter not allowed here

line 230 column 202 - Warning: unescaped & or unknown entity "&task"

line 230 column 217 - Warning: unescaped & or unknown entity "&folderID"

line 258 column 91 - Warning: unescaped & or unknown entity "&d"

line 258 column 154 - Warning: unescaped & or unknown entity "&d"

line 731 column 13 - Warning: <table> lacks "summary" attribute

line 796 column 9 - Warning: <table> lacks "summary" attribute

line 1000 column 9 - Warning: <table> lacks "summary" attribute

Info: Doctype given is "-//W3C//DTD HTML 4.0 Transitional//EN"

Info: Document content looks like HTML 4.01 Transitional

9 warnings, 0 errors were found!

0 errors / 9 warnings

## Comment 1552
Date: 2006-07-01 14:01:19 -0400
From: Filip Jirsák &lt;<Filip.Jirsak@gmail.com>&gt;

Tato chyba je jeden z projevů chyby #527, nejspíš by měla být označena za její
duplikát.

## Comment 1958
Date: 2006-12-19 13:50:18 -0500
From: Robert Krátký &lt;<rk@stickfish.cz>&gt;

duplikát bugu 527

*** This bug has been marked as a duplicate of 527 ***
